vkcs/internal/services/monitoring/requests.go

```diff
@@ -35,11 +36,14 @@ func CreateUser(client *gophercloud.ServiceClient, projectID string, opts Create
 }
 
 func DeleteUser(client *gophercloud.ServiceClient, projectID string, userID string) (r gophercloud.Result) {
+	resp, err := client.Delete(fmt.Sprintf("%s/%s", serviceUserURL(client, projectID), userID), &gophercloud.RequestOpts{
+		MoreHeaders: map[string]string{
+			"Content-Type": "application/json",
```

Comment:
It's kind of strange behavior, why is content type required when there is no request body.

Reply:
wait for fix from cloud monitoring team

Reply:
fixed